src/cmd/cli/command/commands_test.go

```diff
@@ -35,11 +35,12 @@ func (m *MockSsmClient) DeleteParameters(ctx context.Context, params *ssm.Delete
 
 type mockFabricService struct {
 	defangv1connect.UnimplementedFabricControllerHandler
-	canIUseResponse defangv1.CanIUseResponse
+	allowedToUseProvider bool
+	canIUseResponse      defangv1.CanIUseResponse
 }
 
 func (m *mockFabricService) CanIUse(ctx context.Context, canUseReq *connect_go.Request[defangv1.CanIUseRequest]) (*connect_go.Response[defangv1.CanIUseResponse], error) {
-	if !m.canIUseResponse.Gates["provider"] {
+	if !m.allowedToUseProvider {
 		return nil, connect_go.NewError(connect_go.CodePermissionDenied, errors.New("no access to use aws provider"))
```

Comment:
Can we improve this error message?
```suggestion
		return nil, connect_go.NewError(connect_go.CodePermissionDenied, errors.New("your account does not permit access to use the aws provider. upgrade at https://portal.defang.dev/pricing"))
```